debezium-assembly-descriptors/pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>io.debezium</groupId>
         <artifactId>debezium-parent</artifactId>
-        <version>0.8.0-SNAPSHOT</version>
+        <version>0.9.0-SNAPSHOT</version>
```

Comment:
Odd you had to change it back. I'll take a look at what's wrong and exclude this part, as we're going to stay on 0.8.0-SNAPSHOT on master for the time being.